lib/configs/-private/ember/testing.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   plugins: ['ember'],
 
   rules: {
+    "ember/no-pause-test": "error",
```

Comment:
can you add `lerna-changelog` to future releases?

Reply:
Sure in next pr :)

Reply:
thanks